docs/changelog.rst

```diff
@@ -15,6 +15,7 @@ Added
 - New setup method for the **KsatHorFrac** parameter **setup_ksathorfarc** to up-downscale existing ksathorfrac maps. `PR #255 <https://github.com/Deltares/hydromt_wflow/pull/255>`_
 - new function **setup_pet_forcing** to reproject existing pet data rather than computing from other meteo data. PR #257
 - Workflow to compute brooks corey c for the wflow layers based on soilgrids data, soilgrids_brooks_corey. PR #242
+- **setup_lulcmaps** for wflow_sediment: if planted forest data is available, it can be used to update the values of the USLE C parameter. PR #234
```

Comment:
```suggestion
- **setup_lulcmaps** for wflow_sediment: if planted forest data is available, it can be used to update the values of the USLE C parameter. `PR #234 <https://github.com/Deltares/hydromt_wflow/pull/234>`_
```

PR link? I notice that we sometimes do and sometimes don't.

Reply:
So we should be consistent but in hydromt we only put the PR number and not the link anymore. Let's do that in the release PR.

There are two pages where the changes appear:

- [release tag](https://github.com/Deltares/hydromt_wflow/releases/tag/v0.5.0): there if you just put the PR number the link appear automatically but the rst link really does not.
- [changelog doc](https://deltares.github.io/hydromt_wflow/latest/changelog.html#v0-5-0-february-2024): there the links do not appear.

What do you think is best? I think for the release, we generate the note automatically from the changelog so I don't know if we can update the links easily unless we decide to generate it manually again? If not then we should just put the Pr number and not the link.

Reply:
The site we can regenerate. The links in the release when can just edit afterwards. But maybe it becomes a bit of a hassle  so we could opt to leave the link out.